bin/utils/print-joblog-to-file.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project. 2020
+#######################################################################
+
+# Function: Print a JES job log to a USS file
+# Inputs  - jobname and jobid of job to be printed
+# Outputs - a USS file containing the job log, which will be placed in $jobname-$jobid.log
+#         - The job will be purged from the JES spool
+# Uses    - The TSO OUTPUT command
+
+# identify this script
+SCRIPT_DIR="$(dirname $0)"
+SCRIPT="$(basename $0)"
+echo script $SCRIPT started from $SCRIPT_DIR
```

Comment:
Why is is echo needed? is it just debug?

Reply:
It's informational.

Reply:
For who?

Reply:
It's for the person who invokes `zowe-support.sh`, to inform them that `print-joblog-to-file.sh` has started.  That invoker already sees many messages emitted, which are also appended to SUPPORT_ARCHIVE_LOG.  To be fair,  `print-joblog-to-file.sh` is the only script that  `zowe-support.sh` calls, but I'm happy to adopt Zowe house style.